webservice/dispatcher.php

```diff
@@ -35,6 +35,11 @@
 Context::getContext()->container = ContainerBuilder::getContainer('webservice', _PS_MODE_DEV_);
 Context::getContext()->currency = Context::getContext()->currency ?? new Currency(Configuration::get('PS_CURRENCY_DEFAULT'));
 
+// If the currency is not defined in the context, initialize with the default one !
+if (null === Context::getContext()->currency) {
+    Context::getContext()->currency = new Currency(Configuration::get('PS_CURRENCY_DEFAULT'));
+}
```

Comment:
This modification is done at line 36

Reply:
Could you remove it ?

Reply:
Doh ! This piece of code was for another PR … I remove that …